### DISCUSSION TITLE
Plugin.json update

### DISCUSSION THREAD

**Discussion by ddramireddy** [grafana/grafana-json-datasource#138] [Q&A]
Hi @marcusolsson,

I am planning to use your plugin to call our REST API. Our API works based on JWT token. Initially, we need to call Signin API to get the JWT Token and use that token to call the other API. This token needs to be renewed for every 30 mins. 

As per following link, I updated plugin.json and added the routes. However, On restarting grafana, I am getting the error "Failed to start plugin. Plugin file's signature has been modified versus manifest". Is it not allowed to update "plugin.json"? If so, what is the alternative to add tokenAuth section?

https://www.bookstack.cn/read/grafana-v6.2/99ade7a5e84896a8.md

**Comment by marcusolsson** [comment]
The error you're getting is because you're modifying a signed version of the plugin. If you remove the MANIFEST.txt file, you'll get rid of the warning. Though the plugin won't be signed so you'd have to allow it in your grafana.ini.

**Comment by pattersongp** [comment]
Hi @ddramireddy would you be able to share your patch of the plugin that implements this? I'm interested in doing something similar.
